nginx/nginx.conf

```diff
@@ -0,0 +1,21 @@
+http {
+    # Add caching later
+    # proxy_cache_path /data/nginx/cache levels=1:2 keys_zone=endpoint_cache:10m max_size=1g inactive=60m use_temp_path=off;
+    # proxy_cache endpoint_cache;
+    # proxy_cache_valid 200 10m;  # Cache OK (200) responses for 10 minutes
+    # proxy_cache_use_stale error timeout updating http_500 http_502 http_503 http_504;
+    # proxy_set_header Host $host;
+    # proxy_set_header X-Real-IP $remote_addr;
+    # proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    # proxy_set_header X-Forwarded-Proto $scheme;
+
+    server {
+        listen 80;
+
+        location /predictions {
```

Comment:
instead let's do location / 

it will pass everything to http://sceta-server:8001/

Reply:
Done